### DISCUSSION TITLE
Firebird Trigger support empty

### DISCUSSION THREAD

**Discussion by adaliszk** [dbeaver/dbeaver#103] [Q&A] [answered]
I see now the Trigger group in my firebird connections but I did not get any triggers if I open it. How can I give some more debug info about it?

**Comment by serge-rider** [comment, marked as answer]
There are two types of triggers - database (ON CONNECT, etc) and table triggers (ON UPDATE/INSERT,etc). Table triggers lives inside the table element.
Are you sure your database has database triggers?
To validate you can check queries DBeaver executes during metadata reading (in Query Manager view).

**Comment by adaliszk** [comment]
Aaah, that was the trick :D I just used to see all trigger in the main tree in flamerobin :)
